NSS-Day-Cohort-49/tabloid-cli-lionfish
# Add Get All Tags to TagRepo

Modified files TagManager, TagRepository to implement Get All Tags method.
​
Steps to Review
  -Checkout this branch locally.
  -git fetch --all
  -git checkout kk-taglist
  -Open a new Terminal tab (⌘T) and navigate to the server directory.
  -Test app functionality.
  -In app menus, choose (5) Tag Management, and in submenu choose (1) List Tags to see tag keywords in db.
- View code file.
- Confirm file modifications are present as indicated above.
  
- Test regressive code.

Thanks!
